### PR TITLE
Test button component

### PR DESCRIPTION
**Pull request by AlexOverbeck**
#### Update go-tester karma config
This removes kjhtml from the karma config because that node module
has been removed.

#### Adds tests for <go-button> component
This adds basic unit tests around buttons. It additionally tests
some of the basic side effects that the data has on the go-button
template.

### CODE REVIEW

File: projects/go-lib/src/lib/components/go-button/go-button.component.spec.ts [hunk 1]
```diff
@@ -24,4 +24,145 @@ describe('GoButtonComponent', () => {
...
   });
+
+  describe('clicked()', () => {
+    it('sets isProcessing to false if useLoader is false', () => {
+      expect(component.isProcessing).toBe(false);
+      component.clicked();
+      expect(component.isProcessing).toBe(false);
+    });
+
+    it('sets isProcessing to true if useLoader is true', () => {
```
Comment (grahamhency) [inline projects/go-lib/src/lib/components/go-button/go-button.component.spec.ts]: We could probably simplify this to just test that `isProcessing === useLoader` right?

File: projects/go-lib/src/lib/components/go-button/go-button.component.spec.ts [hunk 2]
```diff
@@ -24,4 +24,145 @@ describe('GoButtonComponent', () => {
...
+    });
+
+    it('returns an object that set go-button--positive to true based on buttonVariant', () => {
+      expect(component.classObject()['go-button--positive']).toBe(false);
+
+      component.buttonVariant = 'positive';
+      expect(component.classObject()['go-button--positive']).toBe(true);
+    });
+
+    it('returns an object that set other modifiers to true', () => {
```
Comment (grahamhency) [inline projects/go-lib/src/lib/components/go-button/go-button.component.spec.ts]: Should these be separated out into individual tests?

Comment (AlexOverbeck) [reply projects/go-lib/src/lib/components/go-button/go-button.component.spec.ts]: Meh, I didn't really want to have a bunch of tests to test individual keys in the object. We could separate them in the future if we would like.

File: projects/go-lib/src/lib/components/go-button/go-button.component.spec.ts (hunk 1 above)
Comment (grahamhency) [reply projects/go-lib/src/lib/components/go-button/go-button.component.spec.ts]: I see you updated the assertions in this test to reflect `isProcessing === useLoader`, but should we update the description to reflect this updated test?

File: projects/go-lib/src/lib/components/go-button/go-button.component.spec.ts [hunk 3]
```diff
@@ -24,4 +24,145 @@ describe('GoButtonComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  describe('clicked()', () => {
+    it('sets isProcessing to false if useLoader is false', () => {
```
Comment (grahamhency) [inline projects/go-lib/src/lib/components/go-button/go-button.component.spec.ts]: Do we need this test now that we've got the one below it?

Comment (AlexOverbeck) [reply projects/go-lib/src/lib/components/go-button/go-button.component.spec.ts]: Yep, I wanted to test things in both cases.

**Pull request merged**